infra/modules/service/main.tf

```diff
@@ -12,11 +12,14 @@ locals {
   task_executor_role_name = "${var.service_name}-task-executor"
   image_url               = "${data.aws_ecr_repository.app.repository_url}:${var.image_tag}"
   hostname                = var.hostname != null ? [{ name = "HOSTNAME", value = var.hostname }] : []
+  sendy_api_key           = var.sendy_api_key != null ? [{ name = "SENDY_API_KEY", value = var.sendy_api_key }] : []
+  sendy_api_url           = var.sendy_api_url != null ? [{ name = "SENDY_API_URL", value = var.sendy_api_url }] : []
+  sendy_list_id           = var.sendy_list_id != null ? [{ name = "SENDY_LIST_ID", value = var.sendy_list_id }] : []
```

Comment:
⭐  &nbsp; _(very much optional, just nice to have)_ the `!= null` check seems non-ideal since, to the best of my understanding, we want the sendy credentials to always be defined. I would rather the container blow up if they're undefined, rather than them being empty.

Unless of course I'm totally missing something here ^^

Reply:
The intention of the `!= null` is that the backend api ecs containers do not need these variables, but they both use the same service module. This would be the equivalent of saying if this is on the frontend, then set these values, if it's on the api, set an empty list. 
Is there a cleaner way to do that here?